### ISSUE TITLE
Enter pre-enabled state when engaging at standstill with brake pressed

### ISSUE THREAD

**Issue by adeebshihadeh** [commaai/openpilot#25425]
In this case, we'll need to allow brake actuation to prevent creeping and maintain standstill. However, steering and gas should be blocked. Requires enforcement in both openpilot and panda safety.

**Comment by adeebshihadeh** [comment]
@sshane this still relevant?

**Comment by sshane** [comment]
Yes, the safety isn't merged: https://github.com/commaai/panda/pull/1123
